src/actions/actions.js

```diff
@@ -317,7 +317,7 @@ export function createNewNotebookOnServer(options = { forkedFrom: undefined }) {
   };
 }
 
-export function saveNotebookToServer() {
+export function saveNotebookToServer(appMsg = true) {
```

Comment:
here's the easiest way to turn off the snackbar (without `updateAppMessages` no snackbar)

Reply:
Is this part relevant anymore? i.e. is there any situation where we'd pop up a snackbar? I guess it might be useful for #1586, in which case we could leave it in for now (along with the keybinding).